lib/iris/analysis/maths.py

```diff
@@ -563,7 +563,7 @@ def power(data):
 
     else:
 
-        def power(data, out=None):
+        def power(data, out=None):  # type: ignore[misc]
```

Comment:
```suggestion
        def power(data, out=None):
```

Add `out=None` to the other function on like `561`.